certified-connectors/Ordnance Survey Places/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2985 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "OS Places",
+    "description": "The OS Places API provides a detailed view of an address and its life cycle. It contains all the records of AddressBase\u00ae Premium and so provides all the information relating to an address or property from creation to retirement. It contains local authority, Ordnance Survey and Royal Mail\u00ae addresses, current addresses, and alternatives for current addresses, provisional addresses (such as planning developments) and historic information.",

```

Comment:
@quickjc1 Can you please remove unicodes from the description and if you have any in the swagger or apiProperties.josn file as unicodes may not be supported in all the Power Automate pages.

Reply:
@sriyen-msft yes no problem